modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -6,9 +6,9 @@
 [id="persistent-storage-csi-vsphere-install-issues_{context}"]
 = Removing a third-party vSphere CSI Operator Driver
 
-{product-title} 4.10 includes a built-in version of the vSphere CSI Operator Driver that is supported by Red Hat. If you have installed a vSphere Container Storage Interface (CSI) Driver provided by the community or another vendor, updates to the next major version of {product-title}, such as 4.11, might be disabled for your cluster.
+{product-title} 4.12 includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat. If you have installed a vSphere CSI driver provided by the community or another vendor, updates to the next major version of {product-title}, such as 4.13, or later, might be disabled for your cluster.
 
-{product-title} 4.10 clusters are still fully supported, and updates to z-stream releases of 4.10, such as 4.10.z, are not blocked, but you must correct this state by removing the third-party vSphere CSI Driver before updates to next major version of {product-title} can occur. Removing the third-party vSphere CSI driver does not require deletion of associated persistent volume (PV) objects, and no data loss should occur.
+{product-title} 4.12 clusters are still fully supported, and updates to z-stream releases of 4.12, such as 4.12.z, are not blocked, but you must correct this state by removing the third-party vSphere CSI Driver before updates to next major version of {product-title} can occur. Removing the third-party vSphere CSI driver does not require deletion of associated persistent volume (PV) objects, and no data loss should occur.
```

Comment:
TBH, I'm not sure the expected behavior in coming 4.12, but I think the update is ok to keep the consistency with previous releases.
CC @gcharot to take a look.

Reply:
yes that's the envisioned behavior. we are planning to block 4.12 to 4.13 upgrades if we detect a third party CSI driver.